exercises/sublist/example.rb

```diff
@@ -0,0 +1,18 @@
+
+def sublist(set_a, set_b)
```

Comment:
I think using the `Set` here doesn't solve the problem entirely because of the following example in the [sublist.md](https://github.com/exercism/x-common/blob/master/sublist.md) file.

`A = [1, 2, 4], B = [1, 2, 3, 4, 5], A is not a superlist of, sublist of or equal to B`

If we use the `Set`:

```ruby
a = Set.new([1, 2, 4])
b = Set.new([1, 2, 3, 4, 5])
a.subset?(b)
true # But as the problem specified, it should not be considered a subset.
```